src/components/ui.js

```diff
@@ -145,19 +145,24 @@ AFRAME.registerComponent('ui', {
     this.onModelLoaded = this.onModelLoaded.bind(this);
     this.onStrokeStarted = this.onStrokeStarted.bind(this);
     this.toggleMenu = this.toggleMenu.bind(this);
+    this.open = this.open.bind(this);
+    this.close = this.close.bind(this);
   },
 
-  tick: function () {
-    // Hack until https://github.com/aframevr/aframe/issues/1886
-    // is fixed.
-    this.el.components['ui-raycaster'].refreshObjects();
+  tick: function (time, delta) {
```

Comment:
Since we got ride of ANIME here we no longer need the time and delta parameters.